### PR TITLE
Fixing some typos. Thanks to Fossies for the report

### PR DESCRIPTION
**Pull request by kaleb-himes**

### CODE REVIEW

File: wolfssl/wolfcrypt/port/Renesas/renesas-tsip-crypt.h [hunk 1]
```diff
@@ -42,7 +42,7 @@ typedef enum {
     tsip_Key_RSA1024 = 4,
     tsip_Key_RSA2048 = 5,
     tsip_Key_tls_Rsa2048 = 6,
-    UNKNOW = -1,
```
Comment (embhorn) [inline wolfssl/wolfcrypt/port/Renesas/renesas-tsip-crypt.h]: Is "UNKNOW" used anywhere by Renesas?

Comment (dgarske) [reply wolfssl/wolfcrypt/port/Renesas/renesas-tsip-crypt.h]: I checked and as far as I can tell its not. @miyazakh can you confirm?

Comment (dgarske) [reply wolfssl/wolfcrypt/port/Renesas/renesas-tsip-crypt.h]: We should consider removing the enum if it's not used. Very likely to have conflicts with "UNKNOWN" already defined/used.

Comment (miyazakh) [reply wolfssl/wolfcrypt/port/Renesas/renesas-tsip-crypt.h]: Sorry. I missed the comment. I will take care of "UNKNOWN".

**Pull request merged**